src/Fantomas.Core/CodePrinter.fs

```diff
@@ -1834,15 +1828,36 @@ let genMultilineFunctionApplicationArguments (argExpr: Expr) =
     | _ -> genExpr argExpr
 
 let genTuple (node: ExprTupleNode) =
+    let wrapInfixAppRhsInParenIfNeeded expr =
```

Comment:
Maybe add some comments on why we do this.
The `IfNeeded` doesn't really give the why away.